src/products.js

```diff
@@ -0,0 +1,64 @@
+import renderSandwich from '../src/render-sandwich.js';
+import sandwiches from '../src/data/sandwiches.js';
+
+const regCookieButton = document.getElementById('reg-button');
+const chocCookieButton = document.getElementById('choc-button');
+const cinCookieButton = document.getElementById('cin-button');
+const grahamCookieButton = document.getElementById('graham-button');
+const showAllCookieButton = document.getElementById('show-all-button');
+const listedSandwiches = document.getElementsByTagName('li');
+
+
+renderFlavorList();
+
+
+regCookieButton.addEventListener('click', () => {
+    revealFlavorByClass('regular-cookie');
+});
+
+chocCookieButton.addEventListener('click', () => {
+    revealFlavorByClass('chocolate-cooke');
```

Comment:
Forgot an 'i' here!

Reply:
fixed it! Thanks. Just evolved this code in class and didn't actually check it.